AccelLabrinth/AccelLabrinth/GameViewController.swift

```diff
@@ -0,0 +1,66 @@
+//
+//  GameViewController.swift
+//  AccelLabrinth
+//
+//  Created by Predko Brown Marya C. on 3/1/18.
+//  Copyright © 2018 Darren/Marya. All rights reserved.
+//  Darren Added Accelerator code as it needs an UIViewController
+
+import UIKit
+import SpriteKit
+import GameplayKit
+import CoreMotion
+
+class GameViewController: UIViewController {
+    //needs to be made to allow accelorometer
+    let motionManager = CMMotionManager()
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        if let view = self.view as! SKView? {
+            // Load the SKScene from 'GameScene.sks'
+            if let scene = SKScene(fileNamed: "GameScene") {
```

Comment:
Creating your game scene by making an instance of itself, altho the method you use will work, it doesn't know its a GameScene vs SKScene.

```
let scene = GameScene(size: view.frame.size)
```

This way you will have access to custom functions and variables unique to your GameScene.